### PR TITLE
Add jshint

### PR DESCRIPTION
**Pull request by tofumatt**
Add jshint support for PRs on Travis.

### CODE REVIEW

File: .jscsrc [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "disallowSpacesInAnonymousFunctionExpression": {
```
Comment (tofumatt) [inline .jscsrc:2]: Happy to add more rules, these are just what I pulled from localForage.

File: recroom.js [hunk 2]
```diff
@@ -71,17 +73,18 @@ if (command === 'new' || command === 'create') {
...
+    // The --cordova argument allows users to create a cordova
+    // structure afterward.
+    if (opts.cordova) {
+        // Create the cordova app and directory structure.
+        scaffoldCommand += '&& cordova create --link-to dist dist-cordova ' +
+                           '-i com.yourcompany.yourface -n ' + projectName;
     }
 
-    shell.exec(cmdToExec, function(code, output) {
+    // TODO: Walk through commands in an array instead of relying on &&.
```
Comment (fwenzel) [inline recroom.js:86]: yes.

File: Gruntfile.js [hunk 3]
```diff
@@ -21,7 +37,8 @@ module.exports = exports = function(grunt) {
 
     require('load-grunt-tasks')(grunt);
 
-    grunt.registerTask('default', ['build', 'watch']);
+    grunt.registerTask('default', ['test']);
```
Comment (fwenzel) [inline Gruntfile.js:40]: so default would be if you merely type "grunt"?

**Comment by fwenzel** [comment]
r+

File: Gruntfile.js (hunk 3 above)
Comment (tofumatt) [inline Gruntfile.js:40]: ## Oops, copy paste error. 

tofumatt (Sent from mobile)

Comment (tofumatt) [inline Gruntfile.js:40]: Oh wait, I misread this in the email. Yeah, this is what happens when you type `grunt`.

**Pull request merged**